infra/app.py

```diff
@@ -8,17 +8,37 @@
 
 from aws_cdk import core
 from datalake.datalake_stack import DataLakeStack  # pylint: disable=import-error
+from datalake.networking_stack import NetworkingStack  # pylint: disable=import-error
+from datalake.processing_stack import ProcessingStack  # pylint: disable=import-error
 
 ENV = os.environ.get("DEPLOY_ENV", "dev")
 
+
 app = core.App()
-DataLakeStack(
+
+networking = NetworkingStack(
     app,
-    "geospatial-data-lake",
-    stack_name=f"geospatial-data-lake-{ENV}",
-    env={"region": "ap-southeast-2"},
+    "networking",
+    stack_name=f"geospatial-data-lake-networking-{ENV}",
+    env={"region": os.environ["CDK_DEFAULT_REGION"], "account": os.environ["CDK_DEFAULT_ACCOUNT"]},
 )
 
+processing = ProcessingStack(
+    app,
+    "processing",
+    stack_name=f"geospatial-data-lake-processing-{ENV}",
```

Comment:
We can add `ENV` to the stack name within the stack __init__, to avoid effectively passing this value twice.